config.yml.example

```diff
@@ -8,8 +8,17 @@ database:
   user: icingadb
   password: icingadb
 
+# Connection configuration for the Redis where Icinga writes its configuration, state and history.
+# This is the same connection as configured in the 'icingadb' feature of the primary Icinga node.
+# High availability setups require a dedicated Icinga DB instance per primary node,
+# each connected to the Redis that the respective Icinga node writes to.
 redis:
-  address: localhost:6380
+  # Redis host or absolute Unix socket path.
+  host: localhost
+
+  # Redis port. Defaults to '6379' which is the standard Redis port,
+  # but explicitly set to '6380' here as that is the port of the icingadb-redis package.
```

Comment:
Suggestion: "Redis port. The Redis server provided by the icingadb-redis package listens on port 6380. If not specified, the Redis standard port 6379 is used instead."